modules/pipeline/pipengine/reconciler/taskrun/task_loop.go

```diff
@@ -68,10 +68,30 @@ func (tr *TaskRun) handleTaskLoop() error {
 	}
 	rlog.TDebugf(tr.P.ID, tr.Task.ID, "loop break expr %s evaluate result is false, continue loop", expr)
 
+	// reportTaskForLoop report task should be before resetTaskForLoop for avoid missing task info
```

Comment:
reportTaskForLoop report task before resetTaskForLoop to avoid missing task info